plugins/dynamic-plugins-info/src/components/DynamicPluginsInfoContent/DynamicPluginsInfoContent.tsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+
+import { ContentHeader, SupportButton } from '@backstage/core-components';
+
+import { DynamicPluginsTable } from '../DynamicPluginsTable/DynamicPluginsTable';
+
+export const DynamicPluginsInfoContent = () => (
+  <>
+    <ContentHeader title="">
+      <SupportButton title="Support">Some placeholder text</SupportButton>
```

Comment:
@invincibleJai I wonder if I should just take this out at this time, I don't see this being used in other plugins currently

Reply:
yeah let's open an issue to handle it separately if needed

Reply:
Created https://github.com/janus-idp/backstage-plugins/issues/1150 for this